SECURITY.md

```diff
@@ -3,6 +3,15 @@
 
 Join the [kubestellar-security-announce](https://groups.google.com/u/1/g/kubestellar-security-announce) group for emails about security and major API announcements.
 
+## Dependencies Policy
+
+KubeStellar manages its dependencies with the following policy:
+
+- **Automated Updates:** We use [Dependabot](https://github.com/dependabot) to automatically check for and propose updates to dependencies in Go modules, Python requirements, Dockerfiles, and Helm charts. Dependabot PRs are reviewed and merged by maintainers.
+- **Review Process:** All dependency update pull requests are subject to the same review process as other code changes. Maintainers verify that updates do not introduce breaking changes or known vulnerabilities before merging.
```

Comment:
How is the vulnerability check done before merging?

There is a CI workflow (https://github.com/kubestellar/kubestellar/blob/11a92742824ec3aff6a36d0d94030a26a792e0bc/.github/workflows/verify-action-hashes.yaml) that checks that each GitHub Action reference uses the approved commit of that Action.